cluster-provision/okd/hacks/release-4.4

```diff
@@ -179,28 +179,19 @@ index 01264e898..32cc8059d 100644
  		IsCA:      true,
  	}
  
-@@ -181,7 +181,7 @@ func (a *KubeletClientCertKey) Generate(dependencies asset.Parents) error {
- 		Subject:      pkix.Name{CommonName: "system:serviceaccount:openshift-machine-config-operator:node-bootstrapper", Organization: []string{"system:serviceaccounts:openshift-machine-config-operator"}},
- 		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
- 		ExtKeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth},
--		Validity:     ValidityOneDay,
-+		Validity:     ValidityOneYear,
```

Comment:
applied all changes and fixed all conflicts and then recreated a diff file which is this one
so its there already in the code, else it would reject it